net-analyzer/nload/nload-0.7.4-r1.ebuild

```diff
@@ -0,0 +1,32 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit autotools
+
+DESCRIPTION="Real time network traffic monitor for the command line interface"
+HOMEPAGE="http://www.roland-riegel.de/nload/index.html"
+SRC_URI="http://www.roland-riegel.de/nload/${P}.tar.gz"
+
+LICENSE="GPL-2"
```

Comment:
The LICENSE is wrong:

```
 * This program is free software; you can redistribute it and/or modify
 * it under the terms of the GNU General Public License as published by
 * the Free Software Foundation; either version 2 of the License, or
 * (at your option) any later version.

Reply:
Good catch, thank you!
I'll fix and update the PR.